patches/jest-environment-jsdom+25.5.0.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/node_modules/jest-environment-jsdom/build/index.js b/node_modules/jest-environment-jsdom/build/index.js
+index cb5278c..48cab70 100644
+--- a/node_modules/jest-environment-jsdom/build/index.js
```

Comment:
```
Warning: patch-package detected a patch file version mismatch

  Don't worry! This is probably fine. The patch was still applied
  successfully. Here's the deets:

  Patch file created for

    jest-environment-jsdom@25.5.0

  applied to

    jest-environment-jsdom@26.6.2

  At path

    node_modules/jest-environment-jsdom

  This warning is just to give you a heads-up. There is a small chance of
  breakage even though the patch was applied successfully. Make sure the package
  still behaves like you expect (you wrote tests, right?) and then run

    patch-package jest-environment-jsdom

  to update the version in the patch file name and make this warning go away.
  ```

can we bump this against the latest version?

Reply:
right, check, whoops. bumped.